tools/sklearn/train_test_split.xml

```diff
@@ -0,0 +1,115 @@
+<tool id="sklearn_train_test_split" name="Split Dataset" version="@VERSION@">
+    <description>into train and test subsets</description>
+    <macros>
+        <import>main_macros.xml</import>
+    </macros>
+    <expand macro="python_requirements"/>
+    <expand macro="macro_stdio"/>
+    <version_command>echo "@VERSION@"</version_command>
+    <command detect_errors="exit_code"><![CDATA[
+        python '$__tool_directory__/train_test_split.py'
+            --inputs '$inputs'
+            --infile_array '$infile_array'
+            #if $options.shuffle_selection.shuffle not in ['None', 'simple']
```

Comment:
```suggestion
            #if $options.shuffle_selection.shuffle not in ['None', 'simple']:
```

Reply:
I have noticed that having or not having  the `:` here were the same.